roles/test-utils/mining-device/Cargo.toml

```diff
@@ -24,3 +29,6 @@ tracing = { version = "0.1" }
 tracing-subscriber = "0.3"
 sha2 = "0.10.6"
 tokio = "^1.38.0"
+
+[features]
+abort_mining = []
```

Comment:
what is the purpose of this feature?

Reply:
I added this to replicate exactly the MG test, which says that the `mining-device` should not start mining https://github.com/stratum-mining/stratum/pull/1163/files#diff-cc06b104e73cde20ec61e683c5b4eb74e5422641e422652485c1026dd72a7a98R255

In practice I do not think its needed tbh because anyway the mining-device wont be able to open a channel

Reply:
ok so why do we have it? can't we get away with not having this?

Reply:
Removed the redundant flag

Reply:
> Removed the redundant flag

I don't understand, `abort_mining` seems to be still there